vowpalwabbit/core/src/reductions/cb/cb_explore_adf_large_action_space.cc

```diff
@@ -133,6 +133,29 @@ void cb_explore_adf_large_action_space<randomized_svd_impl, spanner_impl>::rando
   _impl.run(examples, shrink_factors, U, _S, _V);
 }
 
+template <typename randomized_svd_impl, typename spanner_impl>
+size_t cb_explore_adf_large_action_space<randomized_svd_impl, spanner_impl>::number_of_non_degenerate_singular_values()
```

Comment:
this could potentially be used to also return even less actions than specified by d